doc/coding/style.md

```diff
@@ -3,3 +3,10 @@
 ## Philosophy
 1. If it's inserting something into the existing classes/functions, try to follow the existing style as closely as possible.
 1. If it's brand new code or refactoring a complete class or area of the code, please follow as Modern C++ of a style as you can and reference the [C++ Core Guidelines](https://github.com/isocpp/CppCoreGuidelines) as much as you possibly can.
+
+## Formatting
+
+- We use [`.clang-format`](https://github.com/microsoft/PowerToys/blob/master/.clang-format) style file to enable automatic code formatting. You can [easily format source files from Visual Studio](https://devblogs.microsoft.com/cppblog/clangformat-support-in-visual-studio-2017-15-7-preview-1/).
+- If you prefer another text editor or have ClangFormat disabled in Visual Studio, you could invoke [`format_sources`](https://github.com/microsoft/PowerToys/blob/master/format_sources.ps1) powershell script from command line. It gets a list of all currently modified files from `git` and invokes clang-format on them.
+Please note that you should also have `clang-format.exe` in `%PATH%` for it to work. The script can infer the path of `clang-format.exe` version which is shipped with Visual Studio if you launch it from the *Native Tools Command Prompt for VS*.
```

Comment:
Didn't work for me:

![image](https://user-images.githubusercontent.com/3206696/70436417-86a4a400-1a89-11ea-9dc8-649bae91bcf5.png)

But I can use it from within VS.

Reply:
> The script can infer the path of `clang-format.exe` version which is shipped with Visual Studio

So `%VCINSTALLDIR%\Tools\Llvm\bin\clang-format.exe` should work for you. Though you shouldn't invoke it directly, since we already have the script for that!

Reply:
I can use it from within VS, so no need for me to run it outside of VS but we need to provide all the info to make sure people don't run into situations where they can't get it to work.
So please add the `%VCINSTALLDIR%\Tools\Llvm\bin\` as the path to have in `%PATH%`